__tests__/app.test.js

```diff
@@ -128,3 +127,59 @@ describe("GET /api/articles", () => {
     });
 });
 })
+describe("GET /api/articles/:article_id/comments ", () => {
+  test("200: Responds with an array of comments for a given article_id", () => {
+    return request(app)
+    .get("/api/articles/9/comments")
+    .expect(200)
+    .then(({ body }) => {
+      const { comments } = body;
+      expect(comments).toHaveLength(2);
```

Comment:
well done for checking the length here to prevent the false pass due to the foreach!